.nengobones.yml

```diff
@@ -50,9 +50,12 @@ setup_py:
   tests_req:
     - coverage>=4.3
     - nengo-extras
-    - pytest>=3.4,<4
+    - pytest>=3.4
```

Comment:
I'm fine either way so it makes sense to be consistent with `nengo-dl` in the absence of any strong preference. Technically installing `nengo[tests]` will install `pytest` anyway so we could handle it that way? I'm not sure if you can put an extra-requires string in the `setup` call, but if so that would reduce some minor duplication.

Reply:
I think this won't necessarily work in the Nengo Loihi case because we use `pytest-plt`, but I reverted this change at least :man_shrugging: